schema/layer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "/layer",
+  "type": "object",
+  "properties": {
+    "layer": { "type": "string" },
+    "count": { "type": "number" },
+    "geometry": { "type": "string" },
```

Comment:
Those are the only three types that Tippecanoe will generate currently. (Tippecanoe is generating `LineString` instead of `Line`, but if `Line` is correct, I'll fix it.) In the discussions with @flippmoke for a revision of the vector tile spec we've talked about the possibility of splines, data grids, and meshes, but none of those is anything more than an idea yet.